### PR TITLE
Move to helper for lambdakzerobuilder too

### PR DESCRIPTION
**Pull request by ddobrigk**

### CODE REVIEW

**Comment by ddobrigk** [comment]
This is just code cleanup / uniformity with respect to the cascadebuilder: here, I now use a CheckAndUpdate helper method too.

File: PWGLF/TableProducer/lambdakzerobuilder.cxx [hunk 1]
```diff
@@ -179,6 +179,19 @@ struct lambdakzeroBuilder {
     return output;
   }
 
+  void CheckAndUpdate(Int_t lRunNumber, Int_t lTimeStamp)
```
Comment (njacazio) [inline PWGLF/TableProducer/lambdakzerobuilder.cxx:182]: Shouldn't the timestamp be a uint_64?

**Review by njacazio — COMMENTED** [review]
Thanks David!

File: PWGLF/TableProducer/lambdakzerobuilder.cxx (hunk 1 above)
Comment (ddobrigk) [reply PWGLF/TableProducer/lambdakzerobuilder.cxx]: You're right! Thanks, changed!

Comment (njacazio) [reply PWGLF/TableProducer/lambdakzerobuilder.cxx]: Maybe also in yesterday's commit then!

Comment (ddobrigk) [reply PWGLF/TableProducer/lambdakzerobuilder.cxx]: Um yes, sure, I changed it in both already just a moment ago :-D

Comment (njacazio) [reply PWGLF/TableProducer/lambdakzerobuilder.cxx]: Ok great!

**Review by njacazio — APPROVED** [review]

**Pull request merged**